app/src/main/java/edu/uc/groupproject/covid19tracker/RetrofitClientInstance.kt

```diff
@@ -19,15 +19,13 @@ object RetrofitClientInstance {
              */
             if (retrofit == null) {
                 val httpClient = OkHttpClient.Builder()
-                httpClient.addInterceptor(object : Interceptor {
-                    override fun intercept(chain: Interceptor.Chain): Response? {
-                        val request: Request =
-                            chain.request().newBuilder().addHeader("x-rapidapi-host", "coronavirus-monitor.p.rapidapi.com")
-                                .addHeader("x-rapidapi-key", "12a8dba6admshd5f767ad7c36e5bp17cb05jsn3c9cee783e36").build()
-                        return chain.proceed(request)
-                    }
-                })
-                retrofit = retrofit2.Retrofit.Builder()
+                httpClient.addInterceptor { chain ->
+                    val request =
+                        chain.request().newBuilder().addHeader("x-rapidapi-host", "coronavirus-monitor.p.rapidapi.com")
+                            .addHeader("x-rapidapi-key", "12a8dba6admshd5f767ad7c36e5bp17cb05jsn3c9cee783e36").build()
+                    chain.proceed(request)
+                }
+                retrofit = Retrofit.Builder()
```

Comment:
A lambda here makes the code more readable. https://kotlinlang.org/docs/reference/lambdas.html

Reply:
Hmmm, good catch here.